src/main/java/io/anserini/py4j/PyseriniEntryPoint.java

```diff
@@ -131,6 +131,27 @@ public String getRawDocument(String docid) throws Exception {
     return indexUtils.getRawDocument(docid);
   }
 
+  public List<String> getAllSentences(String query, int numHits) throws Exception {
+    Map<String, Float> docScore = search(query, numHits);
+    Map<String, Float> sentencesMap = new LinkedHashMap<>();
+    TokenizerFactory<CoreLabel> tokenizerFactory = PTBTokenizer.factory(new CoreLabelTokenFactory(), "");
+    List<String> allSentences = new ArrayList<String>();
+
+    for (Map.Entry<String, Float> doc : docScore.entrySet()) {
+      List<Sentence> sentences = indexUtils.getSentDocument(doc.getKey());
+      for (Sentence thisSent : sentences) {
+        List<CoreLabel> tokens = tokenizerFactory.getTokenizer(new StringReader(thisSent.text())).tokenize();
+        String tokenizedAnswer = tokens.stream()
+                .map(CoreLabel::toString)
+                .collect(Collectors.joining(" "));
```

Comment:
But java is perfectly capable of storing a `List<List>` ... this would also stop having the re-split the terms on the other end. If this is because of a limitation in the python/java interop, then *document* it, so that dumb people like me don't get confused.